Correct cloud service definition

cloudmelon opened the pull request:
Correct cloud service definition. Cloud service can act as PaaS to host application, and in classic model ( which still existing in some customer's project ) is mandatory while deploying a Azure classic VM ( Cloud service , one VNET etc )

VSC-Service-Account commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/guide/technology-choices/compute-overview.md](https://github.com/cloudmelon/architecture-center/blob/patch-1/docs/guide/technology-choices/compute-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/guide/technology-choices/compute-overview?branch=pr-en-us-625) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C4%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201807041653178432-625%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hGxZtRykFV8VcLMeQ7grb0Rr4mruOis9HeTXeYe5zsA%3D&st=2018-07-04T16%3A49%3A31Z&se=2018-08-04T16%3A54%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

VSC-Service-Account commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/guide/technology-choices/compute-overview.md](https://github.com/cloudmelon/architecture-center/blob/patch-1/docs/guide/technology-choices/compute-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/guide/technology-choices/compute-overview?branch=pr-en-us-625) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C5%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201807050755223531-625%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tXdyh6ldHVIoAL0TEfNOLBOAMfKAS2JqXx1pvS1LXZc%3D&st=2018-07-05T07%3A51%3A37Z&se=2018-08-05T07%3A56%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

cloudmelon commented:
@MikeWasson  Thanks for your kindly remind. I updated my pull-request with the preference of Microsoft. Where I updated is for introduction purpose, it would be better to avoid confusion for readers :)

MikeWasson commented:
I'm going to leave this doc the way it's currently written, which matches the documentation for [Cloud Services](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-choose-me):

> Azure Cloud Services provides PaaS, not infrastructure as a service (IaaS)
